lgc/builder/BuilderImpl.h

```diff
@@ -820,6 +820,13 @@ class SubgroupBuilder : virtual public BuilderImplBase {
   // Create a subgroup mbcnt.
   llvm::Value *CreateSubgroupMbcnt(llvm::Value *const mask, const llvm::Twine &instName) override final;
 
+  // Create a loop merge.
+  llvm::Value *CreateLoopMerge(llvm::Value *const mergeBlock, llvm::Value *const continueBlock,
+                               const llvm::Twine &instName) override final {
+    assert(false && "should never be reached");
```

Comment:
If this is something that is never intended to reach LGC, it should not be an LGC Builder op. Instead you can just make up your own "intrinsic" call name and add it with CreateNamedCall.

Reply:
That makes sense.  I have changed this to `CreateNamedCall("spirv.loop.merge", ...`.